test/physics/base/GeantImporter.test.cc

```diff
@@ -25,23 +24,18 @@ using celeritas::PDGNumber;
 class GeantImporterTest : public celeritas::Test
 {
   protected:
-    void SetUp() override
-    {
-        root_filename_
-            = this->test_data_path("physics/base", "geant-exporter-data.root");
-    }
-
-    std::string   root_filename_;
-    GeantImporter importer_;
+    void SetUp() override {}
+
+    std::string testPath_  = "/Users/4s2/celeritas-project/celeritas/build";
+    std::string root_file_ = testPath_ + "/app/geant-exporter-data.root";
```

Comment:
I think you might have accidentally overridden the changes I made? I forgot to tell you that I pushed a change set aa1d9b80811483544784f5dce6477c1f370f66ed which provides a machine-independent way to access test data.

Reply:
Ok, that is really odd... Anyway, it's now again as it was in https://github.com/celeritas-project/celeritas/commit/aa1d9b80811483544784f5dce6477c1f370f66ed.